CHANGELOG.md

```diff
@@ -106,7 +106,8 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - winbareos.nsi: fix working directory in configure.sed [PR #1288]
 - core: BareosDb::FindLastJobStartTimeForJobAndClient: take into account Running job [PR #1265] [BUG #1466]
 - backup.cc: fail backup when `Write Bootstrap` to pipe fails [PR #1296]
-- webui: fix pool link in job details formatter [PR 1306] [BUG #1489]
+- webui: fix pool link in job details formatter [PR 1303] [BUG #1489]
```

Comment:
```suggestion
- webui: fix pool link in job details formatter [Issue #1489][PR #1303]
```

Reply:
you're right, I will take care of this one and a previous bogus entry I've made.